packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/AccountPanel/ContractNew/ContractMenu.tsx

```diff
@@ -110,31 +99,17 @@ export const ContractStatusSelect: React.FC<ContractStatusSelectProps> = ({
     <>
       <Menu>
         <MenuButton
-          maxW={'auto'}
-          color={`${statusColorScheme[status]}.800`}
-          borderColor={`${statusColorScheme[status]}.800`}
-          bg={`${statusColorScheme[status]}.50`}
+          className={`flex items-center max-h-5 text-${statusColorScheme[status]}.800} border-${statusColorScheme[status]}.800} bg-${statusColorScheme[status]}.50}`}
```

Comment:
Those interpolations might not work correctly since tw compiles at build time all classes ahead of runtime computations. The right way would be to define the classes statically as a whole. 

https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Reply:
what I mean is instead of `text-${statusColorScheme[status]}.800` you need to have `text-gray-800` clearly defined so the compiler can see it.

Reply:
also the interpolation will result in `text-gray.800` which is not going to work in tw. it needs to be `text-gray-800`.